Packs/CommonScripts/Scripts/CVECVSSColor/CVECVSSColor.yml

```diff
@@ -0,0 +1,18 @@
+commonfields:
+  id: cvss_color
+  version: -1
+name: cvss_color
+comment: This dynamic automation parses the CVSS score of a CVE and presents it in the layout in color according to its score.
+script: ''
+type: python
+tags:
+- dynamic-indicator-section
+enabled: true
+scripttarget: 0
+subtype: python3
+runonce: false
+dockerimage: demisto/python3:3.10.11.59070
+runas: DBotWeakRole
+fromversion: 6.0.0
```

Comment:
```suggestion
fromversion: 6.8.0
```

Reply:
It's 6.0.0 on purpose :)
I want it to be available for older versions as well.

Reply:
changed it anyway